spit/classifier.py

```diff
@@ -53,12 +54,73 @@ def init_session(self):
     self.y_train = keras.utils.to_categorical(list(self.label_dict.values()), 
                                               num_classes=len(self.label_dict))
     # add optimizer, learning rate, and loss function
-    self.model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy'], rate=1e-4)
+    adam = optimizers.Adam(lr=0.001, beta_1=0.9, beta_2=0.999, epsilon=None, decay=0.0, amsgrad=False)
+
+    self.model.compile(loss='categorical_crossentropy', optimizer=adam, metrics=['accuracy'])#, rate=1e-4)
     return
   
+    
+  # This method might need to be modified if the comparison of accuracies
+  # is done while training
+  def save_model(self, file_name=None, test_dset=None, test_labels=None, comparing=False):
```

Comment:
Kindly separate out the test part of this into its own method, e.g.
`test()`.  And I like the comparing part, but best to have that be
a separate method too.  Thanks.